### PR TITLE
Add Language and Punctuation design guidelines

### PR DESCRIPTION
**Pull request by andrewhillcode**
intellihr/engineering/guilds/front-end#14395

**The following MUST be checked prior to approval. If not relevant to your PR, remove the line from your description.**
- [ ]  The `styleguide.config.js` has been updated to show examples/new functionality for the component
- [ ]  Test Coverage for any new or updated functionality
- [ ]  New and updated components have been tested locally in lapis and SPA and work as expected
- [ ]  This PR has been tagged with PATCH, MINOR, or MAJOR.
- [ ]  The component has data-component-type and data-component-context attributes following the standard
- [ ]  You have requested a cross-team review on this component so everyone knows it exists

### CODE REVIEW

**Review by wyvl — APPROVED** [review]

File: docs/Language.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+The following guidelines for writing copy are intended to create a consistent style and voice for the intelliHR product.
+
+## Language
+* **Keep text short and simple** to make text accessible to people of all backgrounds
+    * **Do**: This person doesn't have a job record
```
Comment (KaitlynPickard) [inline docs/Language.md:5]: What's our stance on doesn't vs does not?

File: docs/Language.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    * **Update**: When a user is editing a system object
+    * **Exceptions**: Occasionally there will be more context appropriate action words such as `Set Goal` and `Issue Task` which are preferred to the above. The appropriate destructive action word should still be used.
+    * **Page Titles and Submit Buttons**: For titles and buttons, use the following format `[Action Name] [System Object Name]`. For example: `Create Person`, `Add Phone Number`, `Set Goal`. On edit forms the submit button should always be `Save`
+    * **Don't**: Use the word `new` on creation actions. It is implied.
+
+## Capitalization and Punctuation
+* **Use Title Case for buttons, page titles, menu items, and labels** - most words are capitalized
+    * **Example**: Set a Goal
+    * **Do Capitalize**: The first letter of nouns, adjectives, verbs, adverbs, and pronouns
+    * **Do Capitalize**: The first and last words of the text element
```
Comment (KaitlynPickard) [inline docs/Language.md:40]: Are you saying we'd have text like:

'Hello this is my Button'

Seems kinda odd

File: docs/Language.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+* **Use Sentence Case for descriptions, help text, and anything conversational** - most words are lowercase
+    * **Example**: You haven't completed any training in the last quarter.
+    * **Do**: Only capitalize the first word of the text element, as well as any proper nouns.
+
+* **Explain acronyms at the first instance on the page** - when using acronyms
+    * **Example**: Performance Improvement Plan (PIP)
+
+* **Use serial (Oxford) commas** for clarity in lists
+    * **Do**: The Performance Improvement page features all of your skills, goals, and training records
+    * **Don't**: The Performance Improvement page features all of your skills, goals, and training records
```
Comment (KaitlynPickard) [inline docs/Language.md:53]: you did the same thing in the Do and Don't here

Comment (andrewhillcode) [reply docs/Language.md:53]: just dont use lists

Comment (andrewhillcode) [reply docs/Language.md:53]: or do

Comment (andrewhillcode) [reply docs/Language.md:53]: LOL Good catch

File: docs/Language.md (hunk 2 above)
Comment (andrewhillcode) [reply docs/Language.md:40]: Well you're never going to have a button actually labelled that but it would be something like `Hello This is My Button` or `Hello This Is My Button`. The general rule with title text is that everything is caps except for articles and short prepositions - maybe I can reword this to make it make more sense. This rule is in cases where you would start or end the the label with an article/preposition like `A Goal` instead of `a Goal`. 

Also since your example is also more conversational than just being a label (which would just be `My Button`) - you'd be better off with sentence case anyway: `Hello this is my button`

File: docs/Language.md (hunk 1 above)
Comment (andrewhillcode) [reply docs/Language.md:5]: Ah i forgot to copy the part about contractions. Google's conversation API thing prescribes that contractions should always be used because `cannot` and `do not` sound much harsher than `can't` and `don't`

Comment (KaitlynPickard) [reply docs/Language.md:5]: makes sense!

File: docs/Language.md (hunk 3 above)
Comment (aytee17) [inline docs/Language.md]: Remove comma in don't example

**Review by seanmanson — APPROVED** [review]
